workspace.json

```diff
@@ -41,6 +42,7 @@
     "ett-core-shell": "libs/ett/core/shell",
     "ett-dashboard-routing": "libs/ett/dashboard/routing",
     "ett-dashboard-shell": "libs/ett/dashboard/shell",
+    "ett-landing-page-shell-landing-page": "libs/ett/landing-page/shell/landing-page",
```

Comment:
issue: Use `nx generate remove ett-landing-page-shell-landing-page` to delete this library.

Reply:
@LayZeeDK - Thanks and done.